content/javascript/concepts/window-prompt/window-prompt.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'prompt()'
+Description: 'Displays a dialog with an optional message asking users to input some text'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`prompt()`** is a method serviced globally by the window object provided by the user's browser.
```

Comment:
```suggestion
The **`prompt()`** method is serviced globally by the `window` object provided by the user's browser. It displays a dialog with an optional message asking users to input some text.
```
The information in the description also needs to be contained in the start of the entry.